platform/ethereum/blockbook/model_extension.go

```diff
@@ -24,6 +26,21 @@ func (t *Transaction) FromAddress() string {
 	return ""
 }
 
+func (t *Transaction) GetFee() string {
+	status, _ := t.EthereumSpecific.GetStatus()
+	if status != blockatlas.StatusPending {
+		return t.Fees
+	}
```

Comment:
can you add test for this specific function?

Reply:
I think it's already covered in `TestNormalizePage`, we have both pending and completed txs